Fix ASUS Tinker Board link

MichaIng opened the pull request:
It seems the non-localised link is broken.

There is actually a dedicated Tinker Board website: https://tinker-board.asus.co,/
However it presents the new Tinker Edge (not compatible) at first so I chose to link to the product overview where one can see both supported models, Tinker Board and Tinker Board S.

An alternative would be to stay with the old site but UK-localised version, which is still online: https://www.asus.com/uk/Single-Board-Computer/Tinker-Board/
But this might be online for long, also some link on that site are broken, e.g. TinkerOS download leads to 404 😄.

What you think, go like that?
_______
Additionally I removed the verbose flag from link check, so only failing links are listed which makes it much easier to find those and verify whether it's a false alarm of indeed needs to be fixed (like here).

StephanStS commented:
I propose https://tinker-board.asus.com/index.html.

StephanStS reviewed (commented):
My proposal: https://tinker-board.asus.com/index.html

MichaIng commented:
I'm not sure why the SBC names in those download tables are links. Due to `target="_blank"` they did open a new tab with the same page. I removed now target="_blank" so that this links have no effect, but probably those should not be some at all. Also I removed the `target="_blank"` on the download links. I don't like it when there is a new empty tab opened to start the download from and closed directly, seems like an unnecessary overhead to me, and this now fits the main website as well.

Generally the manufacturer links should be aligned, some are currently Markdown syntax, some are HTML syntax. HTML has the benefit that we can add `target="_blank"` to open such external link in a new tab (here it makes sense) and we can add `rel="noopener"` to secure the origin tab/window. Here an article about it, although in German language (sorry Petru 😅): https://www.christoph-freyer.at/blog/links-absichern-mit-rel-noopener/
Not sure if this can be achieved with Markdown syntax as well or if this only allows links to open in the same tab, which is then not good for external links, IMO.
